lib/pages/forward/forward_view.dart

```diff
@@ -31,29 +31,53 @@ class _ForwardViewState extends State<ForwardView> {
     });
   }
 
+  Future<void> _forwardMessage(Map<String, dynamic> message, Room room) async {
```

Comment:
move all the logic function to controller. Only keep some functions: style, building widget, navigation in View class